polling/frequent/workflow.go

```diff
@@ -0,0 +1,30 @@
+package frequent
+
+import (
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+func FrequentPolling(ctx workflow.Context) (string, error) {
+	logger := workflow.GetLogger(ctx)
+
+	ao := workflow.ActivityOptions{
```

Comment:
These timeouts are too short. I would change them to StartToClose=24 hours and Heartbeat=30 seconds. We need to update the Java sample as well.